services/repository-managers/src/main/java/org/sagebionetworks/repo/manager/portals/PortalManagerImpl.java

```diff
@@ -114,7 +115,7 @@ public AccessControlList updatePortalAcl(UserInfo user, String portalId, AccessC
 		// Makes sure the user is not revoking their own permissions
 		PermissionsManagerUtils.validateACLContent(acl, user, Long.valueOf(portalId));
 		
-		if (!user.isAdmin()) {
+		if (!AuthorizationUtils.isPortalManagerOrAdmin(user)) {
```

Comment:
if there is a method that calls the utility, we can spy and check the method is called.